docs/execution_environments.md

```diff
@@ -0,0 +1,61 @@
+# Execution Environments
+
+All jobs use container isolation for environment consistency and security.
+Compliant images are referred to as Execution Environments (EE)s.
+For more information, see [ansible-runner EE docs](https://ansible-runner.readthedocs.io/en/latest/execution_environments.html)
+for information on how container isolation works, and see [ansible-builder docs](https://ansible-builder.readthedocs.io/en/latest/index.html)
+for instructions on how to build them.
+
+The Execution Environment model has an `image` field for the image identifier which will be used by jobs.
+The job details view will link to the execution environment that the job uses.
+
+## Creating and using EEs
+
+Users with and organization's `execution_environment_admin_role` can create new EEs in that organization.
+The RBAC rules follow standard rules for org-scoped resources.
+
+Only superusers can create global EEs (null organization).
```

Comment:
Do we consistently use 'superuser' or 'system administrator' for this user role?

Reply:
hrmm...

```
(env) [alancoding@alan-red-hat awx]$ grep -r "superuser" docs/
docs/auth/ldap.md:"is_superuser": "cn={{superusergroupname}},cn=groups,cn=accounts,dc=example,dc=com"
docs/credentials/custom_credential_types.md:* Superusers (and only superusers) should be able to define custom `Credential
docs/workflow.md:  * Workflow job templates can only be accessible by superusers ---- system admin, admin of the same organization and system auditor and auditor of the same organization with read permission only.
docs/workflow.md:  * Workflow jobs can be deleted by superusers and org admins of the organization of its associated workflow job template, and no one else.
docs/execution_environments.md:Only superusers can create global EEs (null organization).
(env) [alancoding@alan-red-hat awx]$ 
(env) [alancoding@alan-red-hat awx]$ 
(env) [alancoding@alan-red-hat awx]$ grep -r "system admin" docs/
docs/workflow.md:  * Workflow job templates can only be accessible by superusers ---- system admin, admin of the same organization and system auditor and auditor of the same organization with read permission only.
(env) [alancoding@alan-red-hat awx]$ 
(env) [alancoding@alan-red-hat awx]$ grep -r "System admin" docs/
docs/auth/oauth.md:- System admins will be able to see and manipulate all applications in the system;
(env) [alancoding@alan-red-hat awx]$ grep -r "System Admin" docs/
docs/auth/oauth.md:- The System Administrator is able to see and manipulate every token in the system;
docs/clustering.md:Instance Groups can be created by posting to `/api/v2/instance_groups` as a System Administrator.
docs/rbac.md:want a System Administrator to have access to everything that an Organization
docs/prompting.md:but only with Organization or System Administrator privileges.
```

Prefer to keep "superuser" because it will be easiest to hit for a search & replace, should we make it consistent later.

Reply:
I personally prefer System Administrator for user-facing docs, but not enough to do more than raise the question.  Carry on.